src/types/string.ts

```diff
@@ -1,9 +1,10 @@
-import {STRules} from '../rules';
 import {StrongType, makeStrong} from '../strong-type';
 
+import {STRules} from '../rules';
+
 export type StrongString = StrongType<string>;
 
-export function makeString(fallback: string, initial?: string | null): StrongString {
+export function makeString(fallback: string, initial: string | null | undefined): StrongString {
```

Comment:
- [ ] Change `| undefined` type on `initial` to use the optional operator instead `?`.